test/unit/commands/smapi/cli-customization-processor-test.js

```diff
@@ -226,7 +226,7 @@ describe('Smapi test - CliCustomizationProcessor class', () => {
                 name,
                 rootName,
                 bodyPath,
-                required: true,
+                required: false,
```

Comment:
could you also add the opposite case in unit test to make sure we cover different cases?